### PR TITLE
Ratings

### PR DESCRIPTION
**Pull request by stutisk**
image
image

### CODE REVIEW

File: components/ratings/ratings.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../docs/components.css">
+    <link rel="stylesheet" href="../../landing page/style.css">
+    <link rel="stylesheet" href="Typography.css">
+    <script src="https://kit.fontawesome.com/7f4b9645a7.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="ratings.css">
+    <title>Typography||GALAXIA</title>
```
Comment (junaidshaikh-js) [inline components/ratings/ratings.html]: I guess you forgot to rename title

File: components/ratings/ratings.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                </div>
+            </div>
+            <!-- content -->
+            <div class="content">
+                <div class="content-title">
+                    Ratings
+                </div>
+                <div class="content-subtitle">
+                    Ratings play an important role in E-commerce website.it gives us an overview on the average ratings
+                    of the products orthe services
```
Comment (junaidshaikh-js) [inline components/ratings/ratings.html]: typo `orthe`

**Review by junaidshaikh-js — APPROVED** [review]
Good work done! Clean code. 

Some mistakes are there related to text, correct them and it is ready to merge.

File: components/ratings/ratings.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    <link rel="stylesheet" href="../../landing page/style.css">
+    <link rel="stylesheet" href="Typography.css">
+    <script src="https://kit.fontawesome.com/7f4b9645a7.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="ratings.css">
+    <title>Typography||GALAXIA</title>
+</head>
+
+<body>
+    <div class="container">
+        <nav class="nav-bar">
```
Comment (Shivanshsharma01) [inline components/ratings/ratings.html:18]: It's good to wrap the main nav in a "header" tag.

**Review by Shivanshsharma01 — APPROVED** [review]
Code is clean with good use of variables. Can use semantic HTML tags as good practice

**Pull request merged**